game.js

```diff
@@ -56,6 +67,23 @@ class Game {
 	}
 
 	// Draw the game screen(s)
+  resetGame() {
+    this.livesRemaining = 0
+		this.guess = []
```

Comment:
If you reset the `guess` is typically not empty... it contains the same number of characters as the puzzle... where blanks are filled-in with `noGuessChar`.  If you make it empty, then the game will think the puzzle is solved.